profile/README.md

```diff
@@ -3,77 +3,59 @@
 
----
+Take a look at "lab-enhanced" models on the [InstructLab Hugging Face](https://huggingface.co/instructlab) page.
 
-## InstructLab is composed of the following projects
+# Get Started with InstructLab
 
-### [Taxonomy](https://github.com/instructlab/taxonomy)
+* Check out the [Community ReadMe](https://github.com/instructlab/community/blob/main/README.md) to get started with using the project. (Contribution guides are included in this document.)  
+* You may wish to read through the [project's FAQ](https://github.com/instructlab/community/blob/main/FAQ.md) to get more familiar with all aspects of InstructLab.
+* You can find all the ways to collaborate with project maintainers and your fellow users of InstructLab beyond GitHub by visitng our [project collaboration](https://github.com/instructlab/community/blob/main/Collaboration.md) page.
 
-The InstructLab method is driven by taxonomies, which are largely created manually and with care. This repository contains a taxonomy tree that will allow you to create models tuned with your data (enhanced via synthetic data generation) using the InstructLab method.
+# Code of Conduct
+Participation in the InstructLab community is governed by our [Code of Conduct](https://github.com/instructlab/community/blob/main/CODE_OF_CONDUCT.md)
 
-* [Repo](https://github.com/instructlab/taxonomy)
-* [Contributing](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md)
+# Quick Links
 
-### [Command-line Interface](https://github.com/instructlab/instructlab)
+## Governance
 
-This command-line interface for InstructLab will allow you to create models tuned with data you provide using the InstructLab method on your laptop or workstation.
+Overview of how InstructLab project [governance](https://github.com/instructlab/community/blob/main/governance.md) operates.
 
-* [Repo](https://github.com/instructlab/instructlab)
-* [Contributing](https://github.com/instructlab/instructlab/blob/main/CONTRIBUTING/CONTRIBUTING.md)
+## Security
 
-> [!IMPORTANT]
-> It is important to understand that running InstructLab on a laptop will give you a low-fidelity approximation of both synthetic data generation (using the `ilab generate` command)
-> and model instruction tuning (using the `ilab train` command, which uses QLoRA.) The quality of the results you get using these tools on a laptop will not be as high-fidelity as they might be using
-> a larger teacher model and a different training method. We have optimized InstructLab to enable community members with modest hardware to be able to use the technique. If you have more sophisticated
-> hardware, you can configure InstructLab to use a larger teacher model [such as Mixtral](https://huggingface.co/docs/transformers/model_doc/mixtral) in order to achieve higher-fidelity results.
```

Comment:
This seems important for setting expectations. Do you feel this is sufficiently covered somewhere else?

Reply:
It is covered in the ilab README hence I think it is both covered and out of place here.